### ISSUE TITLE
SCM accessibility feedback

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210255]
From @jooyoungseo

SCM view needs some commands exposed because atm, you have to focus the SCM view and then tab or use space/arrow keys to get to these.

1. focus commit message
2. focus staged changes (not tabbable)
3. focus changes (not tabbable)

We should add `!screenReaderEnabled` to this keybinding so users can review their commit messages without going up in history
`{
  "key": "up",
  "command": "scm.viewPreviousCommit",
  "when": "scmInputIsInFirstPosition && scmRepository && !suggestWidgetVisible"
}`

**Comment by rzhao271** [comment]
I wasn't able to find any commands about focusing the staged changes. What are some verification steps for this issue?

**Comment by lszomoru** [comment]
Verification steps:
* Setup keybindings for the following commands:
  * workbench.scm.action.focusNextInput
  * workbench.scm.action.focusPreviousInput
  * workbench.scm.action.focusNextResourceGroup
  * workbench.scm.action.focusPreviousResourceGroup
* Verify that the keybindings are working as expected

**Comment by rzhao271** [comment]
It seems like focusNextInput helps focus across repositories (such as in a multi-root workspace with changes in each repo) but that focusNextResourceGroup focuses across resource groups in a single repository.

I initially expected focusNextResourceGroup to skip between repositories, but I think it is more structured to have one command skip between repositories and another command navigate within a repository.

**Comment by lszomoru** [comment]
Correct. The behaviour that you are describing is the intended one.

**Comment by meganrogge** [comment]
@lszomoru, JooYoung was describing that it's confusing to have `next` and `previous` in the command description. We should have a command to focus the current input. cc @jooyounsgeo

**Comment by jooyoungseo** [comment]
Additionally, screen reader users and those who don’t have a visual perception of the Git tree layout need an explicit command to focus on the “Changes” and “Staged Changes” views. While the newly added "go to next" and "previous resource group" commands might serve as a workaround, they operate in a cycle mode rather than focusing directly on the user's intended target.
